ebl_ai/model.py

```diff
@@ -0,0 +1,77 @@
+import tempfile
+from io import BytesIO
+from typing import Union, List, Sequence
+
+import attr
+import numpy as np
+from PIL import Image
+from mmcv import Config
+from mmocr.apis import init_detector, model_inference
+
+
+@attr.s(auto_attribs=True, frozen=True)
+class BoundingBoxesPrediction:
+    top_left_x: float
+    top_left_y: float
+    width: float
+    height: float
+    probability: float
+
+
+class Model:
+    def __init__(self, configFile: str, checkpoint: str):
+        self.model = init_detector(
+            Config.fromfile(configFile), checkpoint=checkpoint, device="cpu"
+        )
+
+        if self.model.cfg.data.test["type"] == "ConcatDataset":
+            self.model.cfg.data.test.pipeline = self.model.cfg.data.test["datasets"][
+                0
+            ].pipeline
+
+    def _predict(self, image_path: str) -> List[List[float]]:
+        """
+        :param image_path: str
+        :return: List[List[float]] of vertices of polygons with last
+        entry beeing the score (probability between 0 - 1)
+        """
```

Comment:
Having a docstring on a "protected" method doesn't feel very useful. Most of the content is obvious from the signature. If the return value is too obscure, maybe add a sane model for it.